scripts/test_all.wren

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env wrenc
+
+import "io" for Directory, File
+import "os" for Process
+import "meta" for Meta
+import "essentials" for Strings
+
+// 1. set cwd to wren-testie root
+
+// TODO (enhancement): static method File.tempfile()
+// TODO (enhancement): reading stdout of a process
+
+var tmpfile = ".git.root.%(Process.pid)"
+Process.exec("sh", ["-c", "git rev-parse --show-toplevel > %(tmpfile)"])
+var rootdir = File.read(tmpfile).trim()
+File.delete(tmpfile)
+
+Process.chdir(rootdir)
+
+import "../testie" for Testie
+import "../src/expect" for Expect
+
+// 2. fetch all wren files
+var testDir = "./tests/"
+var testFiles = Directory.list(testDir)
+                         .where {|f| Strings.globMatch(f, "*.wren")}
+                         .map {|f| testDir + f[0..-6]}
```

Comment:
Is this just stripping extension?

Reply:
Yes.

Reply:
We need a nicer way, but we have no `basename` yet. :)